### PR TITLE
Add support for CI envs in colour support check

### LINKED ISSUE #61: Support for CI environments is lacking
Current implementation doesn't reliably determine colour support in CI environments, resulting e. g. in CI for https://github.com/pinojs/pino-pretty/tree/feat/colorette breaking.

I think these lines from `supports-colour` should be included:

```
	if ('CI' in env) {
		if (['TRAVIS', 'CIRCLECI', 'APPVEYOR', 'GITLAB_CI', 'GITHUB_ACTIONS', 'BUILDKITE', 'DRONE'].some(sign => sign in env) || env.CI_NAME === 'codeship') {
			return 1;
		}

		return min;
	}
```

### PR DESCRIPTION
**Pull request by kibertoad**
fixes #61

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
+
+const isCompatibleTerminal = (process.stdout != null &&
+    process.stdout.isTTY &&
+    env.TERM &&
+    env.TERM !== "dumb")
+
+const isCI = ('CI' in env && ['CIRCLECI', 'GITLAB_CI', 'GITHUB_ACTIONS'].some(sign => sign in env))
+
+let enabled = (!isDisabled) &&
+  (isForced || isWindows || isCompatibleTerminal || isCI)
```
Comment (jorgebucaran) [inline index.js:15]: Can we do without the destructuring assignment (I know it's a bit strange, but we technically still support pre Node 6) and format the expression to this:

```js
const env = process.env

const isDisabled = "NO_COLOR" in env
const isForced = "FORCE_COLOR" in env
const isWindows = process.platform === "win32"

const isCompatibleTerminal =
  process.stdout != null &&
  process.stdout.isTTY &&
  env.TERM &&
  env.TERM !== "dumb"

  const isCI =
  "CI" in env &&
  ("GITHUB_ACTIONS" in env || "GITLAB_CI" in env || "CIRCLECI" in env)

let enabled =
  !isDisabled && (isForced || isWindows || isCompatibleTerminal || isCI)
```

**Review by jorgebucaran — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (kibertoad) [reply index.js]: Done!

**Review by jorgebucaran — APPROVED** [review]

File: index.js (hunk 1 above)
Comment (jorgebucaran) [reply index.js]: Thank you!

**Comment by jorgebucaran** [comment]
@kibertoad Just checking, but does it make sense to automatically enable color inside a CI? I thought we'd want to disable it instead. Maybe I'm having a time-lapse.

**Comment by kibertoad** [comment]
@jorgebucaran It makes reading logs in CI console easier, and also this is a default behaviour for `support-colours`, so I would say that this is what the majority of users are actually expecting.

**Comment by jorgebucaran** [comment]
Gotcha. 👍

**Pull request merged**